kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -294,6 +307,11 @@ define([
                         this.updateView();
                     }.bind(this));
 
+                    $('td:eq(4)', nRow).find('button[data-download]').off('click').on('click', function (e) {
```

Comment:
If you set the click function as an arrow function, you don't need to bind to this - it always goes to `this`'s context.
Also, semicolon.
```suggestion
                    $('td:eq(4)', nRow).find('button[data-download]').off('click').on('click', (e) => {
```

Reply:
(really I just wanted to play with the new suggestion feature. 😁 )